README.livemd

```diff
@@ -0,0 +1,235 @@
+# Req
+
+## Section
+
+[![CI](https://github.com/wojtekmach/req/actions/workflows/ci.yml/badge.svg)](https://github.com/wojtekmach/req/actions/workflows/ci.yml)
+[![License](https://img.shields.io/hexpm/l/req.svg)](https://github.com/wojtekmach/req/blob/main/LICENSE.md)
+[![Version](https://img.shields.io/hexpm/v/req.svg)](https://hex.pm/packages/req)
+[![Hex Docs](https://img.shields.io/badge/documentation-gray.svg)](https://hexdocs.pm/req)
+
+[![Run in Livebook](https://livebook.dev/badge/v1/blue.svg)](https://livebook.dev/run?url=https%3A%2F%2Fgithub.com%2Fwojtekmach%2Freq%2Fblob%2Fmain%2FREADME.livemd)
```

Comment:
Adds a "Run in Livebook" button.

The rest of the changes are minor changes to the code samples to make them work in Livebook.